### ISSUE TITLE
[Bug] - _embedded is enabled even without any HAL configurations

### ISSUE THREAD

**Issue by alesaudate** [spring-projects/spring-hateoas#1218]
Given the following piece of code:

```
 @GetMapping
    public CollectionModel<EntityModel<Driver>> listDrivers(@RequestParam("page") int page) {
        Page<Driver> driverPage = driverRepository.findAll(PageRequest.of(page, PAGE_SIZE));

        List<EntityModel<Driver>> driverList = new ArrayList<>();
        for (Driver driver : driverPage.getContent()) {
            EntityModel<Driver> objectEntityModel = new EntityModel(driver);
            driverList.add(objectEntityModel);
        }

        Link lastPageLink = linkTo(methodOn(DriverAPIImpl.class).listDrivers(driverPage.getTotalPages() - 1))
                .withRel("lastPage");

        return new CollectionModel<>(driverList, lastPageLink);
    }
```

It's rendering as result:

```{
    "_embedded": {
        "driverList": [
            {
                "id": 2,
                "name": "string",
                "birthDate": null
            },
            {
                "id": 3,
                "name": "string",
                "birthDate": null
            }
        ]
    },
    "_links": {
        "lastPage": {
            "href": "https://localhost:8080/drivers?page=0"
        }
    }
}
```

I expected it to return something like this:

```
{
	"driverList": [{
			"id": 2,
			"name": "string",
			"birthDate": null
		},
		{
			"id": 3,
			"name": "string",
			"birthDate": null
		}
	],
	"_links": {
		"lastPage": {
			"href": "https://localhost:8080/drivers?page=0"
		}
	}
}
```

As a side note, consider that if I use the same method to add a link to the EntityModel it works as expected. The `_embedded` element just shows up when using `CollectionModel`.  

The whole code is available under https://github.com/alesaudate/rest-v2/tree/master/cap09 .

**Comment by odrotbohm** [comment]
Would you mind not arbitrarily opening tickets that we just closed another one for? This is a duplicate of #1214. We concluded the discussion there, I left a comment on the linked StackOverflow ticket. I can only once again politely ask to not to create work for maintainers in hunting down tickets. We're organizing our work here. Thanks for understanding!
